openehr-rm/src/main/java/com/nedap/archie/rm/composition/Composition.java

```diff
@@ -109,9 +111,8 @@ public void setCategory(DvCodedText category) {
     }
 
     @JsonIgnore
-    public void setCategory(String codePhrase) {
-        this.category = new DvCodedText();
-        category.setDefiningCode(new CodePhrase(codePhrase));
+    public void setCategory(String value, String codePhrase) {
+        this.category = new DvCodedText(value, codePhrase);
```

Comment:
If used, it would result in a coded text with a value missing. So that would be invalid data, which means you need to change your application anyway, so I would say leave this backwards compatible. I'll add  it to the release notes.